### PR TITLE
Add disk encyption key to the google_compute_instance_template - Sovereign support

### PR DESCRIPTION
**Pull request by rune92**
---
**Checklist**

I applicable, I acknowledge that I have:
- [ ] Read the [contributing guide](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md)
- [ ] Ran `terraform fmt` on all modified files
- [ ] Regenerated the relevant README.md files using [`tools/tfdoc.py`](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md#fabric-tools)
- [ ] Made sure all relevant tests pass

### CODE REVIEW

File: get-pip.py
Comment (juliocc) [inline get-pip.py:1]: remove this file

**Review by juliocc — CHANGES_REQUESTED** [review]

**Comment by juliocc** [comment]
Thanks @rune92. This looks good now

Can you sign the CLA?

**Comment by ludoo** [comment]
@rune92 can you accept our CLA?

**Comment by rune92** [comment]
> Thanks @rune92. This looks good now
> 
> Can you sign the CLA?

yepp, thanks I accepted. 

Screenshot 2024-12-09 at 09 36 21

**Comment by ludoo** [comment]
> > Thanks @rune92. This looks good now
> > Can you sign the CLA?
> 
> yepp, thanks I accepted.

there's one commit by `D****-K*****@t-sy****s.com` which is not passing the CLA

**Comment by rune92** [comment]
> > > Thanks @rune92. This looks good now
> > > Can you sign the CLA?
> > 
> > 
> > yepp, thanks I accepted.
> 
> there's one commit by `D****-K*****@t-sy****s.com` which is not passing the CLA

Alright, then I signed the CLA with D****-K*****@t-sy****s.com email as well. Can you check now?

**Review by juliocc — APPROVED** [review]

**Pull request merged**